esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -3612,7 +3612,15 @@ void CWsDfuEx::setDFUQueryFilters(IEspDFUQueryRequest& req, StringBuffer& filter
     setFileNameFilter(req.getLogicalName(), req.getPrefix(), filterBuf);
     setFileTypeFilter(req.getFileType(), filterBuf);
     WsDFUHelpers::appendDFUQueryFilter(getDFUQFilterFieldName(DFUQFFattrowner), DFUQFTwildcardMatch, req.getOwner(), filterBuf);
-    WsDFUHelpers::appendDFUQueryFilter(getDFUQFilterFieldName(DFUQFFkind), DFUQFTwildcardMatch, req.getContentType(), filterBuf);
+
+    if (req.getInvertContent_isNull() || !req.getInvertContent())
+    {
+        WsDFUHelpers::appendDFUQueryFilter(getDFUQFilterFieldName(DFUQFFkind), DFUQFTwildcardMatch, req.getContentType(), filterBuf);
```

Comment:
question/not new: if ContentType is not set - why is building/adding it as a filter ?

Reply:
@jakesmith the current logic was not changed except for honoring the invert content flag. The appendDFUQueryFilter method simply returns if there is no filter to add.  IMO if that logic should be changed, a separate Jira should be opened.

Reply:
>The appendDFUQueryFilter method simply returns if there is no filter to add. 

ok, that's fine.